middlewares/articlesMiddleware.js

```diff
@@ -51,7 +51,7 @@ export default class AriclesMiddleware {
       const foundArticle = await articleHelpers.findArticleByAuthorId(authorId, title);
       if (foundArticle && foundArticle.title === title) {
         return Response.send(
-          res, 400, [], 'an article with that title already exist', 'failed',
+          res, 403, [], 'an article with that title already exist', 'failed',
```

Comment:
#### Nitpick
- Thought there was a constants file for status codes

Reply:
Hello Audax. The constants file requires the status code as parameters. What he's done is to pass it the required parameters.